src/veritysetup/veritysetup.c

```diff
@@ -81,6 +120,12 @@ static int run(int argc, char *argv[]) {
                         return 0;
                 }
 
+                if (argc > 7) {
+                        flags = parse_flags(argv[7]);
+                        if (r < 0)
```

Comment:
this doesn't look right - it's fixed in a later commit, and those changes should instead be squashed back into the first commit